app/services/GuaranteeBalanceService.scala

```diff
@@ -71,27 +82,33 @@ class GuaranteeBalanceService @Inject() (actorSystem: ActorSystem,
   }
 
   def pollForGuaranteeBalance(balanceId: BalanceId)(implicit
-    hc: HeaderCarrier
+    hc: HeaderCarrier,
+    request: DataRequest[_]
   ): Future[Either[HttpResponse, BalanceRequestResponse]] = {
+    logger.info("[GuaranteeBalanceService][pollForGuaranteeBalance]")
```

Comment:
A bit picky but can we have a little message in here to accompany the file/method names